test/python/WMCore_t/Services_t/CRIC_t/CRIC_t.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+"""
+Test case for CRIC
+"""
+from __future__ import print_function
+
+import unittest
+
+from WMCore.Services.CRIC.CRIC import CRIC
+from WMQuality.Emulators.EmulatedUnitTestCase import EmulatedUnitTestCase
+
+class CRICTest(EmulatedUnitTestCase):
+    """
+    Unit tests for SiteScreening module
+    """
+
+    def __init__(self, methodName='runTest'):
+        super(CRICTest, self).__init__(methodName=methodName)
+
+    def setUp(self):
+        """
+        Setup for unit tests
+        """
+        super(CRICTest, self).setUp()
+        self.myCRIC = CRIC()
+
+    def testWhoAmI(self):
+        print("XXXXX")
+        print(self.myCRIC.whoAmI())
```

Comment:
This returns empty list, when does it going to be populated in CRIC

Reply:
It has been populated already. From what I discussed with Stefano, it seems it does not work when we add a proxy to the request header. When we make the request with the service certificate (different pem for cert and key), that works just fine(?)